db/entity/DiscordServerConfig.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  DeleteDateColumn,
+  Index,
+  Relation,
+} from "typeorm";
+import type { StarkyModuleConfig } from "../../starkyModules/types";
+import { DiscordServer } from "./DiscordServer";
+
+@Entity()
+@Index(["id", "DiscordServerId", "deletedAt"], {
+  unique: true,
+  where: '"deletedAt" IS NOT NULL',
+})
+@Index(["id", "DiscordServerId"], {
+  unique: true,
+  where: '"deletedAt" IS NULL',
+})
+export class DiscordServerConfig {
+  @PrimaryGeneratedColumn("increment")
+  id: string;
+
+  @Column()
+  DiscordServerId: string;
+
+  @Column()
+  starknetNetwork: string;
+
+  @Column()
+  discordRoleId: string;
+
+  @Column()
+  starkyModuleType: string;
+
+  @Column("jsonb", { nullable: false, default: {} })
+  starkyModuleConfig: StarkyModuleConfig;
+
+  @ManyToOne((type) => DiscordServer, (server) => server.serverConfigs)
+  DiscordServer: Relation<DiscordServer>[];
```

Comment:
Same for the uppercase.

Also, I think it's wrong here. It's the right type of relationship (Many to One, i.e. many server configs to one server), but the type is weird `Relation<DiscordServer>[]`

To me this should be:

```suggestion
  discordServer: DiscordServer;
```